packages/cli/internal/pkg/cli/configure_format.go

```diff
@@ -56,14 +56,14 @@ func BuildConfigureFormatCommand() *cobra.Command {
 		Short: "Sets default format option for output display of AGC commands. Valid format options are 'text' and 'table'",
 		Args:  cobra.ArbitraryArgs,
```

Comment:
Should this be `Args:  cobra.ExactArgs(1),` to ensure the single argument passed. or is there a case where we would want to have more than one arg passed to this command?

Reply:
The feedback from pervious PRs suggested to move away from `Args: cobra.ExactArgs(1)` to give customers a more readable error when argument is not provided.

Reply:
"Error: accepts 1 arg(s), received 0" only tells the customer the what while "Error: a single format value must be provided" tells the customer the why as well. This means that for a customer to resolve the issue, they don't need to type "-h" to know what the values are.

Ideally, we could populated the valid args like you mention or generate the message with the valid options like we do with some of our other commands.